### PR TITLE
Add authentication documentation

### PR DESCRIPTION
**Pull request by k-svard-ericsson**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -234,6 +234,12 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
 </pre></td></tr></table>
 </p>
 
+<h3>
+<a id="configuration" class="anchor" href="#configuration" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Configuration</h3>
```
Comment (p-backman-ericsson) [inline index.html]: I think we need an example of starting the microservice with authentication. You can of course use dummy values.

**Comment by k-svard-ericsson** [comment]
This configuration is located in the application.yml file which generate always reads when starting up.

**Comment by p-backman-ericsson** [comment]
Where is that documented?

**Comment by k-svard-ericsson** [comment]
I will update the docs to mention the application.yml.

**Pull request merged**